Open Communication Guidelines

xentenza opened the pull request:
- added Open Communication Guidelines to the news footer
- corrected copyright date

xentenza committed:
Open Communication Guidelines

- added Open Communication Guidelines to the news footer
- corrected copyright date

williambelle reviewed (commented):
Pour éviter la duplication du template `footer-legal-news.twig` et `footer-light-news.twig`, je crois que c'est mieux avec un if dans le template `footer-legal.twig`. Par exemple :

```
{% if isNews %}
  <a href="https://go.epfl.ch/charte">Charte</a>
{% endif %}
```

C'est normal qu'on part sur epfl-mediacom.notion.site ? (au lieu de epfl.ch)
Et pas de traduction ?

xentenza committed:
ocg with conditions

- first PR revocated (too many files)
- introduction of conditions
- corrected footer page/news-detail

xentenza commented:
> Pour éviter la duplication du template `footer-legal-news.twig` et `footer-light-news.twig`, je crois que c'est mieux avec un if dans le template `footer-legal.twig`. 

Excellente idée, Voilà qui est fait

> C'est normal qu'on part sur epfl-mediacom.notion.site ? (au lieu de epfl.ch) Et pas de traduction ?
Oui, car le projet est encore en cours de traduction. Mise en ligne sur le site de l'EPFL le 19 avril 2024. Je te tiens au courant avant.

xentenza commented:
@williambelle Les pages derrière les go sont traduites. Je n'ai plus qu'à les inclure dans le menu du site ;-)
